frontend/src/components/supp-allowances/TransportationAllowance.vue

```diff
@@ -222,6 +222,12 @@ export default {
       }
       this.$emit('deleteDocument', documentId)
     },
+    areDocumentsMissing(model) {
+      if (model.monthlyLease == 0) {
+        return model.documentsToUpload?.length === 0 && model.uploadedDocuments?.length === 0
+      }
+      return model.documentsToUpload?.length + model.uploadedDocuments?.length < 2
```

Comment:
why this areDocumentsMissing is different to the one in Summary?

Reply:
It's not different - we just only look at uploadedDocuments in the summary (because any documents will have been uploaded in order to reach that page)

Reply:
I see. thanks Jen!